pom.xml

```diff
@@ -66,33 +71,50 @@
           <target>${maven.compiler.target}</target>
         </configuration>
       </plugin>
-
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-checkstyle-plugin</artifactId>
-        <version>2.17</version>
+        <groupId>com.coveo</groupId>
+        <artifactId>fmt-maven-plugin</artifactId>
+        <version>2.5.1</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>check</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>com.mycila</groupId>
+        <artifactId>license-maven-plugin</artifactId>
```

Comment:
looks like we should remove the license validation in checkstyle then? or do these play nice?

Reply:
oh, you are removing the checkstyle plugin. The fmt plugin more or less seems to use the same style then?

Reply:
FWIW this whole thing was intended for projects that are going to be maintained in github.com/gwtproject, so being a bit opinionated isn't a bad thing.

This proj is not meant to be a sample for how other people should write their own projects and "3rd party" libraries.